src/test/java/com/google/sps/ContextServletTest.java

```diff
@@ -43,15 +48,34 @@
 /** This is a JUnit test for sentiment mockedAnalysis */
 @RunWith(JUnit4.class)
 public class ContextServletTest {
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
   private static final String URL_PARAMETER = "url";
   private static final String TEST_URL = "WkZ5e94QnWk";
   private static final String TEST_TITLE = "Test Title";
   private static final String TEST_CHANNEL = "Test Channel";
+  private static final String REQUEST_INFO = "snippet,statistics";
   private static final int NUM_LIKES_VAL = 10;
   private static final int NUM_DISLIKES_VAL = 1;
 
   @Test
-  public void testContextGet() throws IOException, GeneralSecurityException {
+  public void testYoutubeAPIConstructed() throws IOException, GeneralSecurityException {
+    // Test the constructeExecuteYoutubeReuest method for returning a youtube request
+    YouTube youtubeService =
+        new YouTube(GoogleNetHttpTransport.newTrustedTransport(), JSON_FACTORY, null);
```

Comment:
We probably don't wan't to be constructing a real Youtube object here since its construction may result in some external calls to the youtube API.

Can we change this to use a `mock` instead of a `spy`?